doc/setup-binderhub.rst

```diff
@@ -1,104 +1,147 @@
+:ref:`setup-registry`. We'll copy/paste the contents of this file in the steps
+below.
 
-Make DNS records for both binder and jupyterhub that point to that IP. For
-example, make both::
+We also need two random tokens to configure out BinderHub. Generate these
+tokens by running the following commands then copying the outputs.::
 
-   hub.mydomain.com
-   binder.mydomain.com
-
-point to the same ``EXTERNAL-IP`` address of the ingress controller.
+    openssl rand -hex 32
+    openssl rand -hex 32
 
 .. note::
 
-   When you configure the helm chart, you will add the DNS entry for binder and
-   jupyterhub in the ``config.yaml`` file.
-
+   This command is run **twice** because we need two different tokens.
 
-Configure the Helm Chart
-------------------------
+Create ``secret.yaml``
+----------------------
 
-First create a file called ``secret.yaml``. In it, put the following code::
+Create a file called ``secret.yaml``. In it, put the following code::
 
-    jupyterhub:
+  jupyterhub:
       hub:
-        cookieSecret: "<openssl rand -hex 32>"
         services:
           binder:
-            apiToken: "<openssl rand -hex 32>"
+            apiToken: "<output of FIRST `openssl rand -hex 32`>"
       proxy:
-        secretToken: "<openssl rand -hex 32>"
-    registry:
-      password: |
-        <the json file from Service Accounts>
-    hub:
-      services:
-        binder:
-          apiToken: "<openssl rand -hex 32>"
+        secretToken: "<output of SECOND `openssl rand -hex 32`>"
+  registry:
+    password: |
+      <contents of the json file from Service Accounts>
+  hub:
+    services:
+      binder:
+        apiToken: "<output of FIRST `openssl rand -hex 32`>"
 
 .. tip::
 
-   Don't forget the `|` after the ``password:`` label.
+   The content you put just after ``password: |`` must all line up at the same
+   tab level.
 
 .. tip::
 
-   For more information on getting a registry password, see
-   `setup-registry`_.
+   Don't forget the ``|`` after the ``password:`` label.
+
+Create ``config.yaml``
+----------------------
+
+Create a file called ``config.yaml``. In it, put the following code::
+
+  registry:
+    prefix:  gcr.io/<google-project-id>/<prefix>
+    enabled: true
+
+  rbac:
+     enabled: false
+  jupyterhub:
+     hub:
+        rbac:
+           enabled: false
+
+.. note::
+
+   Note that the ``google-project-id`` in ``prefix:`` is the project *ID*,
+   not just the project name. Sometimes these will be different (if you have
+   created a project name that's been used before).
+
+.. note::
+
+   `<prefix>` can be any string, and will be appended to image names. We
+   recommend something descriptive such as ``dev`` or ``prod``.
+
+Install BinderHub
+-----------------
+
+First grab the latest helm chart for BinderHub.::
+
+    helm repo add jupyterhub https://jupyterhub.github.io/helm-chart
+    helm repo update
+
+Now we'll **install the Helm Chart** using the configuration
```

Comment:
~~Now we'll~~ Next, **install the Helm Chart** ~~using...running~~ by entering the following command::